### PR TITLE
Add Validation with Cmake 3.14

### PR DESCRIPTION
**Pull request by JX278**

### CODE REVIEW

**Review by caic99 — COMMENTED** [review]
Why there is a space in filename [.github/workflows/build_test .yml](https://github.com/deepmodeling/deepflame-dev/pull/63/files/8f114ef5f86fc5ef7576a593b7a9d33b2e5a0d14..5bec1d97b0b91f85d80476c801514ec7de0033cf#diff-5ed1e04b87ad888c40761177b4d8c1ba114d604db341240fa59a75d3ed31c8ae)?

File: .github/workflows/Test.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    steps:
+     - name: Checkout
+       uses: actions/checkout@v3
+     - name: initialize
+       env:
+        DEBIAN_FRONTEND: noninteractive
+       run: |
+        apt install -y --no-install-recommends 
+        apt-get update 
+        apt-get install -y sudo wget unzip git software-properties-common make cmake g++ mpich openmpi-bin libopenmpi-dev libscalapack-mpi-dev
```
Comment (caic99) [inline .github/workflows/Test.yml:19]: Please pay attention to the usage of `--no-install-recommends`.
```suggestion
        apt-get update 
        apt-get install -y --no-install-recommends sudo wget unzip git software-properties-common make cmake g++ mpich openmpi-bin libopenmpi-dev libscalapack-mpi-dev 
```

**Review by caic99 — CHANGES_REQUESTED** [review]

**Comment by JX278** [comment]
> Why there is a space in filename [.github/workflows/build_test .yml](https://github.com/deepmodeling/deepflame-dev/pull/63/files/8f114ef5f86fc5ef7576a593b7a9d33b2e5a0d14..5bec1d97b0b91f85d80476c801514ec7de0033cf#diff-5ed1e04b87ad888c40761177b4d8c1ba114d604db341240fa59a75d3ed31c8ae)?

It's now modified to a version without the space.

**Comment by zhixchen** [comment]
Is this PR still a work-in-progress?

**Comment by JX278** [comment]
> Is this PR still a work-in-progress?

It's now done. The validation of 2D TGV is changed to compare the maximum temperature in the y direction.

**Review by zhixchen — APPROVED** [review]

**Pull request merged**